x/staking/types/validator_test.go

```diff
@@ -334,3 +301,22 @@ func TestBondStatus(t *testing.T) {
 	require.Equal(t, BondStatusBonded, Bonded.String())
 	require.Equal(t, BondStatusUnbonding, Unbonding.String())
 }
+
+func mkValidator(tokens int64, shares sdk.Dec) Validator {
+	return Validator{
+		// TODO: use valAddr1
```

Comment:
should this be addressed?

Reply:
yes, i set it as a reminder. updated now.